### ISSUE TITLE
Project Maintainence

### ISSUE THREAD

**Issue by subtlepseudonym** [rakyll/gotest#21]
This project hasn't been updated in ~6 months and [rakyll](https://github.com/rakyll/) is taking a break from open source. In the meantime, I am happy to maintain this project on [my fork](https://github.com/subtlepseudonym/gotest/) where I will actively review pull requests and merge changes. (And hopefully others will review my changes)

It's not my intention to use this issue as self-promotion. I use this project every day and would like to see it maintained. Either way, I'll continue to keep an eye on the issues and pull requests here.

**Comment by rakyll** [comment]
@subtlepseudonym, would you be interested in taking some maintainership role on this repo?

**Comment by subtlepseudonym** [comment]
Absolutely! Anything I should know beyond what's in the contribution doc?
